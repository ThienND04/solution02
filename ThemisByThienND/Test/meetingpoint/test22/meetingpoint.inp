240 438 26
44 221 32387
163 169 80932
110 75 196963
101 89 640730
119 210 215676
136 47 620187
32 33 660307
188 236 39656
25 37 150022
186 29 78944
17 62 494007
201 122 737804
54 162 511875
230 6 967417
168 63 264888
187 81 2271
136 11 32290
188 65 595964
173 208 221185
130 106 229339
16 115 38209
144 198 887842
153 108 28973
94 67 127723
117 136 359135
141 186 750750
49 127 81527
12 203 798045
179 101 869393
212 151 82578
147 226 638832
155 8 191256
202 170 5032
48 51 131774
43 142 901648
166 187 804797
1 40 841386
34 97 44459
92 187 781216
156 32 1000000
220 32 474096
127 190 18500
110 83 241526
120 193 456402
77 228 244071
73 4 606903
142 11 307024
37 129 122453
171 73 23992
171 217 94420
146 32 794874
138 31 158172
217 24 758121
52 174 199940
137 132 140684
55 14 983033
5 184 858319
29 40 105240
227 114 46942
218 41 299251
64 129 150966
206 136 518467
49 19 145642
236 16 649761
167 198 215982
112 187 27375
233 176 552679
47 2 3147
146 40 28346
1 65 37869
13 95 638476
161 174 60892
95 2 6564
40 31 98053
193 2 47706
25 62 489110
184 153 36639
222 211 300812
85 54 594977
195 33 217105
56 2 906613
79 149 23792
201 149 685328
12 74 868671
100 185 55379
232 78 255547
34 17 31033
187 82 873409
36 152 156856
33 21 61230
229 19 17930
190 34 657942
187 159 928110
62 171 210270
105 197 251127
130 65 576755
53 28 854258
69 103 397239
29 61 60353
157 229 179697
112 160 747110
205 169 654614
194 161 154968
194 215 184529
150 68 65944
66 172 813564
96 22 837995
86 10 88233
50 142 60104
6 2 26038
55 72 820168
5 205 831465
139 111 138270
22 2 88892
209 234 42649
19 38 447354
224 25 879582
113 196 133422
131 97 539271
183 136 580235
52 79 846724
72 139 140430
59 105 848594
197 208 22742
98 2 639161
80 55 656231
22 102 755917
37 196 79403
59 174 668507
125 120 178151
1 221 10755
144 184 16290
114 121 646558
83 124 167328
53 37 121502
184 106 757265
57 230 76827
144 12 15261
6 153 23218
113 114 68109
187 89 611729
225 181 5241
57 180 140036
179 109 144140
99 162 58992
6 80 660614
58 167 204205
74 225 53640
140 132 98317
213 234 45992
79 56 857130
238 71 285689
131 177 259905
176 235 170903
171 153 658422
35 158 70568
101 2 13083
27 171 233669
143 168 410480
237 64 20966
161 64 107850
95 85 698294
51 93 31907
153 191 862876
41 65 628823
32 55 13058
80 153 637396
16 20 18324
165 109 111528
210 46 831475
120 169 263254
40 126 132294
126 128 99545
141 152 782326
207 101 25484
2 118 90543
22 172 24277
17 163 844894
227 199 68638
18 226 246507
129 69 202199
207 239 663004
97 114 635473
194 39 108071
224 110 543450
115 205 51610
57 107 32751
33 240 353553
229 84 4176
48 122 836627
235 130 192123
110 68 103111
176 110 15347
67 103 784797
76 127 179252
59 177 850311
138 59 752705
149 118 17267
164 166 653
161 3 27602
216 215 69340
49 24 585372
18 79 455695
60 43 907350
16 17 19526
5 35 56946
29 136 112007
141 47 60390
228 185 44699
133 176 224644
214 84 302636
27 117 41119
108 227 683081
9 6 65297
78 218 341730
3 175 51360
79 139 941536
208 184 562073
39 113 237722
103 215 612994
227 116 206325
236 8 45069
124 211 244434
217 4 725315
21 38 702544
224 154 700516
84 151 156324
78 220 131951
12 95 47778
19 201 274895
218 78 341729
24 133 815707
189 123 7533
224 1 107740
16 200 957244
59 203 824629
141 216 594719
86 45 201717
75 210 98235
55 90 810686
145 190 229823
81 103 74792
81 46 74299
205 172 419
47 29 732195
29 16 712414
89 155 255199
26 22 520222
38 234 26454
111 229 351109
7 148 419368
116 81 927824
182 88 318723
228 129 228445
62 178 171829
2 27 941348
134 26 160830
55 123 991887
22 30 766010
96 104 178707
92 56 83313
42 221 822545
231 101 984365
194 107 37249
49 135 214312
165 179 32612
168 213 452447
1 183 43902
50 48 45116
73 103 569082
52 170 281454
169 147 654509
155 70 415006
187 1 42886
158 54 820711
219 216 62823
6 79 23445
197 1 626029
42 173 36656
7 124 667065
64 9 773675
205 95 106184
112 70 424546
207 118 51975
74 139 68005
100 2 953169
2 17 3401
231 205 884699
235 53 152056
222 93 240145
200 103 865568
152 57 70764
28 187 61657
73 205 570937
144 218 895538
110 54 540507
234 69 501944
199 70 198666
87 21 205216
180 140 85691
141 121 4271
115 211 419013
174 150 123176
51 184 903306
131 59 590406
206 112 35408
158 170 315838
183 13 601942
22 165 826196
156 144 947756
173 101 856071
129 89 60230
194 75 64776
75 62 349942
24 189 985864
162 187 579671
221 213 45938
105 221 866401
208 183 604871
179 21 143578
32 47 14215
202 170 5030
229 216 156618
115 221 51185
142 103 848043
62 10 172665
66 194 14604
2 1 802
54 4 33097
69 88 304346
163 194 63832
211 193 432445
112 235 736485
142 23 319644
136 119 106576
133 213 803794
74 231 74433
163 36 159288
211 79 430667
10 108 591844
59 90 813347
76 112 820670
46 33 563557
122 218 944341
199 164 153556
6 138 754427
197 219 95854
166 212 102162
98 77 59256
8 151 55616
233 183 39262
32 23 625642
202 112 540264
36 70 195001
183 220 448359
147 12 58510
193 131 570461
224 230 886518
209 4 24235
90 221 831154
182 44 455126
181 12 917070
233 191 829771
187 7 15437
199 62 195263
200 93 40936
76 15 19081
205 167 628715
138 13 135424
54 49 519469
163 165 66792
49 180 147241
32 59 10396
231 175 264047
238 24 600444
28 125 578518
69 48 345583
125 149 608983
1 204 935375
85 195 189916
196 153 866645
114 90 126739
164 40 5644
165 43 854089
71 229 428472
179 35 118684
221 24 34024
81 32 22449
190 17 688975
84 72 344774
7 192 856816
191 135 68469
237 103 771370
56 60 61735
204 88 118381
136 11 32290
88 81 776378
131 198 339281
13 229 143402
104 234 738281
61 81 635178
234 122 10901
47 33 674522
137 70 129274
37 171 128817
141 54 46343
198 22 868554
96 140 64604
223 86 108249
202 125 72536
228 153 893232
47 94 768530
112 231 927989
22 159 881301
90 223 25449
19 108 405479
170 77 83664
218 121 897331
210 194 33459
185 221 887837
151 117 320327
203 224 745451
168 15 362709
3 128 88570
238 37 192074
41 175 67512
214 54 698571
20 234 31354
224 171 600741
60 144 898743
2 234 9899
111 131 234583
153 91 53159
43 95 54434
187 169 725279
58 111 92920
8 217 84342
68 238 109078
978504 848859
68180 575842
311835 198833
599602 73129
86251 188637
910383 452149
901859 287587
245251 678766
737594 226829
12738 780302
446678 237732
338363 208978
944085 761130
506981 757275
589734 972927
360086 319939
449914 110837
184571 302060
201059 7217
725404 191484
42834 617040
233873 765624
755129 713516
516348 107831
428271 538654
725572 831100
