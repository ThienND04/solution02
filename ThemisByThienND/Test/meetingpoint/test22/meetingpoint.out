663884630376
70363259960
182351732630
68901465024
66693177261
414670369390
263748913570
239979755598
208027144190
13145896236
218026394520
191655813580
640532405865
343970892089
400116237246
244307188334
101649721070
125225701799
7317013186
175611891240
44205630348
241362081206
512331617501
98892888410
290568597099
492278109268
